pkgs/development/tools/rust/cargo-show-asm/test-add.nix

```diff
@@ -0,0 +1,19 @@
+# Test the basic usage of cargo-asm on x86_64.
```

Comment:
perhaps making a directory here would be more readable?

Reply:
You mean a sub-directory for the test? Kind of overkill. Maybe renaming it to `test-basic-x86_64.nix` should be fine.

Reply:
> You mean a sub-directory for the test?

yeah

i don't really mind the name, we can keep it as it is